layouts/shortcodes/code-example.html

```diff
@@ -0,0 +1,51 @@
+{{- $pathURL := (print "/content" (.Get "pathURL"))}}
+{{- $files := readDir $pathURL -}}
+<div class="example-tabs">
+    {{ $.Scratch.Set "tabClass" "active" }}
+    {{- range $files }}
+        {{ $filename_arr := split .Name "." }}
+        {{ $filename := index $filename_arr 0}}
+        {{ $filename_ext := index $filename_arr 1 }}
+
+        {{ if eq $filename ($.Get "exampleId") }}
+            <button class="example-tab {{ $.Scratch.Get "tabClass"}} {{$.Get "example-id"}}" id="{{.Name}}">
+            {{ if eq $filename_ext "py"}}
+                Python
+            {{ else if eq $filename_ext "js" }}
+                Javascript
+            {{ else if eq $filename_ext "cs" }}
+                C#
+            {{ else if eq $filename_ext "yaml" }}
+                YAML
+            {{ else if eq $filename_ext "toml" }}
+                TOML
+            {{ else}}
+                .{{ $filename_ext }}
+            {{ end }}
+            </button>
+            <script>
+                !function() {
+                    currentTab = window.parent.document.getElementById("{{.Name}}");
```

Comment:
actually this will still pollute global scope unless you use 
```suggestion
                    var currentTab = window.parent.document.getElementById("{{.Name}}");
```